### PR TITLE
github atom mac cheat sheet

### PR DESCRIPTION
**Pull request by gautamkrishnar**
updated to resolve conflicts with #1933 

-----
IA Page: https://duck.co/ia/view/github_atom_mac_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata post coming soon

File: share/goodie/cheat_sheets/json/github-atom.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
-    "id": "github_atom_cheat_sheet",
+    "id": "github_atom_mac_cheat_sheet",
     "name": "Atom",
-    "description": "A hackable text editor for the 21st Century",
+    "description": "A hackable text editor for the 21st Century. ",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/github-atom.json]: No periods :smiling_imp:

File: share/goodie/cheat_sheets/json/github-atom.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
 {
-    "id": "github_atom_cheat_sheet",
+    "id": "github_atom_mac_cheat_sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/github-atom.json]: the filename also needs to match the id pattern.

we should rename to `atom-mac` if the other file will be `atom.json` or prepend both filenames with `github-`

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=atom cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/github_atom_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/github-atom.json (hunk 1 above)
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/github-atom.json]: Hehehe :+1: :smile:  OK

File: share/goodie/cheat_sheets/json/github-atom.json (hunk 2 above)
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/github-atom.json]: @moollaza :+1: done

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=atom cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/github_atom_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=atom cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/github_atom_cheat_sheet_mobile.png)

**Comment by gautamkrishnar** [comment]
@moollaza Description updated

**Comment by gautamkrishnar** [comment]
@moollaza  Waiting for your review.... :)

**Comment by moollaza** [comment]
@gautamkrishnar Thanks! This LGTM :+1:

File: share/goodie/cheat_sheets/json/github-atom-mac.json [hunk 3]
```diff
@@ -1,14 +1,14 @@
 {
-    "id": "github_atom_cheat_sheet",
+    "id": "github_atom_mac_cheat_sheet",
     "name": "Atom",
-    "description": "A hackable text editor for the 21st Century",
+    "description": "Collection of keyboard shortcuts for atom text editor in mac",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/github-atom-mac.json]: We should capitalize `Atom` here and rephrase `in mac` to `for Mac OS`

Comment (moollaza) [inline share/goodie/cheat_sheets/json/github-atom-mac.json]: or something like "for the Mac OS version of Atom"

File: share/goodie/cheat_sheets/json/github-atom-mac.json [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
     "metadata":{
         "sourceName": "Bugsnag",
         "sourceUrl": "https://bugsnag.com/blog/atom-editor-cheat-sheet"
     },
     "aliases": [
-        "atom",
-        "atom text editor"
+        "atom mac",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/github-atom-mac.json]: we should probably add `atom osx, atom os x`

**Pull request merged**